vpn/tls.go

```diff
@@ -5,12 +5,15 @@ package vpn
 //
 
 import (
-	"crypto/tls"
+	otls "crypto/tls"
 	"crypto/x509"
+	"encoding/hex"
 	"errors"
 	"fmt"
 	"io/ioutil"
 	"net"
+
+	tls "github.com/refraction-networking/utls"
```

Comment:
Do you mind calling this `utls` and leaving `tls` as the system TLS?

Reply:
I was wondering: do you think it makes sense to export tlsFactoryFn and allow users of the library to change it?

I am inclined to say YAGNI (in the miniooni context) and I can better delete defaultTLSFactory for now...

Reply:
Thanks for explaining to me the issue with `utls` and `tls` together. Any deviation from your current diff sounds to me as a premature optimization since we are not using this library with `miniooni` yet. I'd perhaps add a TODO comment in the code explaining that we have some future design choices regarding TLS support.